### ISSUE TITLE
Clicked trackable link from Gmail message not recognized by c:geo

### ISSUE THREAD

**Issue by Vike91** [cgeo/cgeo#8755]
**Describe the bug:**
Clicking the watchlisted trackable link in received Gmail message won't properly open it in c:geo

**To Reproduce:**
Get an email about watchlisted trackable.
Click on the trackable name (as you would similarly click cache name to open it in c:geo)

**Actual behavior/state after performing these steps:**
Received a popup message: `c:geo can't display the trackable you want. Is it really a trackable?`

**Expected behavior/state after performing these steps:**
Trackable details page opens.

**Version of c:geo used:**
2020.08.04

**Is the problem reproducible:**
Yes

**Screenshots:**
Please see video on https://www.youtube.com/watch?v=Tncbcueoo9M

**System information:**
```
--- System information ---
Device: ONEPLUS A6013 (OnePlus6T, OnePlus)
Android version: 10
Android build: ONEPLUS A6013_41_200505
c:geo version: 2020.08.04
Google Play services: enabled - 20.30.16 (120400-323885386)
Low power mode: inactive
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: orientation
Hide caches: own/found 
Hide waypoints: -
HW acceleration: enabled (default state)
System language: en (c:geo forced to English)
System date format: M/d/yy
Debug mode active: no
System internal c:geo dir: /data/user/0/cgeo.geocaching (73.5 GB free) internal
User storage c:geo dir: /storage/emulated/0/cgeo (73.5 GB free) external non-removable
Geocache data: /storage/emulated/0/Android/data/cgeo.geocaching/files/GeocacheData (73.5 GB free) external non-removable
Database: /storage/emulated/0/Android/data/cgeo.geocaching/files/databases/data (42.9 MB) on user storage
Fine location permission: granted
Write external storage permission: granted
Geocaching sites enabled:
   geocaching.com: Logged in (Login OK) / PREMIUM
   extremcaching.com: Logged in (Login OK)
   opencaching.de: Logged in (Login OK)
   opencache.uk: Logged in (Login OK)
   opencaching.nl: Logged in (Login OK)
   opencaching.pl: Logged in (Login OK)
   opencaching.us: Logged in (Login OK)
   opencaching.ro: Logged in (Login OK)
   Geocaching.su: Logged in (Login OK)
Geocaching.com date format: d.M.yyyy
Installed c:geo plugins:  none
--- End of system information ---

```

**Additional context:**
None

**Comment by Lineflyer** [comment]
Maybe related to #4719 ?

**Comment by Vike91** [comment]
Yes. Looks like the link is a coord.info link, which opens fine when you long press the link and open it in browser. Short tapping takes the link straight to c:geo and bounces it back with the aforementioned pop-up message.

For the information, on my phone I have set up all the coord.info links to open directly in c:geo.

coord.info links that link to geocaches are working properly.

**Comment by moving-bits** [comment]
@Vike91 
Can you please provide the specific link which fails in your environment?
(I tried the link in the linked issue #4719, http://coord.info/TB392FV), but that does work here in the emulator.)

**Comment by Vike91** [comment]
(Whoops, sorry, wrong button!)

http://coord.info/TB6VZGZ

**Comment by moving-bits** [comment]
Thanks for the link. It opens fine in c:geo as well (as expected, as it has the same structure), so it looks like GMail is somehow fiddling with the link.

I tried sending myself an e-mail with this link to a gmail test account - but that opens fine in c:geo. Maybe you can forward your e-mail to our support address so we can investigate further what's happening?

**Comment by Vike91** [comment]
Forwarded the email to support email as requested.

**Comment by moving-bits** [comment]
@Vike91 
Thanks for forwarding the e-mail.

@Lineflyer 
Can you forward that e-mail to me via PM? Thanks.

**Comment by Vike91** [comment]
> I tried sending myself an e-mail with this link to a gmail test account - but that opens fine in c:geo. Maybe you can forward your e-mail to our support address so we can investigate further what's happening?

Just a quick question: were you using an Android phone?

**Comment by Lineflyer** [comment]
@moving-bits 
Just forwarded (on your mail provider you might need to check the spam folder...for some reason a typical problem with this provider when sending mails from support).

**Comment by moving-bits** [comment]
Found the reason: Some mail clients add a port number to the URL (`:80` in this case), which we did not check for... I'll create a PR for it.

**Comment by Vike91** [comment]
I can confirm this has been fixed successfully in version 2020.09.02-RC.

Thabk you very much!
